app/keepers/keepers.go

```diff
@@ -298,8 +312,18 @@ func NewAppKeepers(
 		scopedICAControllerKeeper, bApp.MsgServiceRouter(),
 	)
 
+	appKeepers.ICAModule = ica.NewAppModule(&appKeepers.ICAControllerKeeper, &appKeepers.ICAHostKeeper)
 	icaHostIBCModule := icahost.NewIBCModule(appKeepers.ICAHostKeeper)
 
+	appKeepers.RouterKeeper = routerkeeper.NewKeeper(
```

Comment:
new keeper, so the key should be added to the v12 upgrade handler store & any params required